guides/common/modules/proc_filtering-and-removing-facts.adoc

```diff
@@ -0,0 +1,36 @@
+:_content-type: PROCEDURE
+
+[id="proc_optimizing-performance-by-removing-nics-from-database_{context}"]
+= Optimizing performance by removing NICs from database
+
+Filter and exclude the connections using the `Exclude pattern for facts stored in {Project}` and  `Ignore interfaces with matching identifier` option.
+By default, these options are set to a reasonable list for hypervisors such as, `libvirt`, `ovirt`, `RHV`, or `VMWare`.
```

Comment:
```suggestion
By default, these options are set to a reasonable list for hypervisors such as `libvirt`, `ovirt`, `RHV`, or `VMWare`.
```

I don't know if it's a good idea to add this list. Maybe "most common hypervisors" instead?